softlearning/environments/adapters/robosuite_adapter.py

```diff
@@ -0,0 +1,144 @@
+"""Implements a RobosuiteAdapter that converts Robosuite envs into SoftlearningEnv."""
+
+from collections import OrderedDict
+
+import numpy as np
+import robosuite as suite
+import gym
+from gym import spaces
+
+from .softlearning_env import SoftlearningEnv
+from softlearning.environments.adapters.gym_adapter import GymAdapter
+
+ROBOSUITE_ENVIRONMENTS = {}
+
+def convert_robosuite_to_gym_obs_space(robosuite_obs_space):
+    assert isinstance(robosuite_obs_space, OrderedDict)
+    listDict = []
```

Comment:
We usually prefer snake case variable names in python: `list_dict`

Reply:
Btw, Python Enhancement Proposal (PEP 8 specifically) provides official guidelines and for Python styles in general: https://www.python.org/dev/peps/pep-0008/.

I'm not saying that you should read that, but it's good to keep in mind. Linter will do all this automatically for you though :smile: